resources/js/ForVisitors.vue

```diff
@@ -172,6 +172,14 @@
                 <span class="text-base">{{ $t("(duration of performance: 20 min)") }}</span>
             </section>
         </div>
+        <div class="pt-12">
+            <a
+                href="https://labiennale.vivaticket.it/en/tour/la-biennale-di-venezia-57-esposizione-internazionale-d-arte/754"
+                class="font-display py-3 px-8 text-lg border-white/30 border-2 hover:bg-orange hover:border-orange hover:text-blue-darker transition rounded-[70px] text-center"
```

Comment:
```suggestion
                class="font-display py-3 px-8 text-lg border-white/30 border-2 hover:bg-orange hover:border-orange hover:text-blue-darker transition rounded-[70px] text-center" target="_blank"
```

because this is going outside of the web route